README.md

```diff
@@ -305,6 +305,9 @@ The configuration options for the Kafka Connect source connector for IBM MQ are
 | mq.message.mqmd.read                    | Whether to enable reading of all MQMD fields                                                                            | boolean | false          |                                                                                                                                                                                     |
 | mq.max.poll.blocked.time.ms             | How long the connector will wait for the previous batch of messages to be delivered to Kafka before starting a new poll | integer | 2000           | It is important that this is less than the time defined for `task.shutdown.graceful.timeout.ms` as that is how long connect will wait for the task to perform lifecycle operations. |
 | mq.client.reconnect.options             | Options governing MQ reconnection.                                                                                      | string  | ASDEF          | ASDEF, ANY, QMGR, DISABLED                                                                                                                                                          |
+| mq.jms.receive.timeout                  | The timeout in milliseconds for receiving messages from JMS Consumer.                                                   | long    | 2000L          | 1 or greater                                                                                                                                                                        |
+| mq.jms.reconnect.delay.min.ms           | The minimum delay in milliseconds for reconnect attempts.                                                               | long    | 64L            | 1 or greater                                                                                                                                                                        |
```

Comment:
I think there is a risk that users will misunderstand this and confuse it with the JMS client reconnect feature - https://www.ibm.com/docs/en/ibm-mq/9.2?topic=reconnection-using-automatic-jms-client 

It's unfortunate, because the name you've used is absolutely the most obvious and logical name. But I think we should name it differently to make it clear that this is how long the connector will pause after an exception before it tries again. 

(Also, I recommend removing the "L" from the default value, as above)

Reply:
I still think that we need to rename these parameters. I suggest that we don't need to mention "jms" (as that's largely an implementation detail that a user of the Connector doesn't need to know about). 

If you re-frame these parameters as configuring the connection to the queue manager, then we can avoid the overlap with the existing parameter names.